src/script/interpreter.cpp

```diff
@@ -1126,8 +1166,30 @@ bool TransactionSignatureChecker::CheckSig(const vector<unsigned char>& vchSigIn
     return true;
 }
 
-bool TransactionSignatureChecker::CheckLockTime(const CScriptNum& nLockTime) const
+bool TransactionSignatureChecker::CheckLockTime(const CScriptNum& nLockTime, bool fSequence) const
```

Comment:
Why is this being put in the same function anyway?
It only shares 10 lines of code,  and adds another 10.
It could be easily a 10 line function and a 16 line function without an `fSequence` parameter, and no need to have two sets of comments dictating two different use cases.